### PR TITLE
Annualpopulation UI

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1187 

Before:-
image

After:-
image

### CODE REVIEW

File: django_project/population_data/admin.py [hunk 1]
```diff
@@ -6,15 +6,36 @@
...
+    search_fields = [
+        'property_name',
+        'year',
+        'scientific_name',
+        'common_name'
+    ]
     form = AnnualPopulationForm
 
+    def property_name(self, obj: AnnualPopulation):
+        return obj.owned_species.property.name
```
Comment (dimasciput) [inline django_project/population_data/admin.py:31]:
```suggestion
        return obj.property.name
```

File: django_project/population_data/admin.py [hunk 2]
```diff
@@ -6,15 +6,36 @@
...
+
+    def scientific_name(self, obj: AnnualPopulation):
+        return obj.owned_species.taxon.scientific_name
```
Comment (dimasciput) [inline django_project/population_data/admin.py:34]:
```suggestion
        return obj.taxon.scientific_name
```

File: django_project/population_data/admin.py [hunk 3]
```diff
@@ -6,15 +6,36 @@
...
+
+    def common_name(self, obj: AnnualPopulation):
+        return obj.owned_species.taxon.common_name_varbatim
```
Comment (dimasciput) [inline django_project/population_data/admin.py:37]:
```suggestion
        return obj.taxon.common_name_varbatim
```

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 42 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 12 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 43 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 57 sec

**Pull request merged**